e2e/tests/myatb.e2e.ts

```diff
@@ -0,0 +1,184 @@
+import {device, by, element, expect} from 'detox';
+import {findTextViewElement, goToPage, scrollToElement, scrollToElementText} from "../utils/common";
+import {skipOnboarding} from "../utils/onboarding";
+import {addFavouriteLocation, deleteFavouriteLocation, editFavouriteLocation, sortLabelExists} from "../utils/myatb";
+import setLocation from "../utils";
+
+describe('My AtB', () => {
+    beforeAll(async () => {
+        await device.launchApp({
+            permissions: {
+                location: 'inuse',
+            },
+            languageAndLocale: {
+                language: 'en',
+                locale: 'US',
+            },
+        });
+        await setLocation(62.4305, 9.3951);
+        await skipOnboarding();
+
+        await goToPage('My AtB')
+    });
+
+    beforeEach(async () => {
+        //await device.reloadReactNative();
+        await goToPage('My AtB')
+        await element(by.type('RCTCustomScrollView')).scrollTo('top')
+    });
+
+    // TODO How to check change of appearance?
```

Comment:
Godt spørsmål 🤔 Potensielt mulig å kjøre `device.captureViewHiarchy()` og tyde fila som blir lagret. Tror det er mer arbeid enn det er verdt dog. En annen crazy mulighet er `device.takeScreenshot()` og tyde fargene med ImageMagick eller noe. Igjen, mer arbeid enn det er verdt. En tredje og kanskje mer oppnåelig plan er å printe ut fargekoder under "Design system"-viewet og sjekke at det er endret.

Reply:
Gode tips! Tenker jeg kan ta inn disse kommentarene for å se på dette senere. Men tror samtidig at akkurat denne testen er mindre verdt enn mange som ikke er tatt med enda mtp kost vs nytte.